_tuning-your-cluster/availability-and-recovery/remote-store/remote-cluster-state.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: default
+title: Remote Cluster State
+nav_order: 5
+parent: Remote-backed storage
+grand_parent: Availability and recovery
+---
+
+# Remote Cluster State
+
+Introduced 2.10
+{: .label .label-purple }
+
+Remote Cluster State protects against any cluster state metadata loss resulting due to quorum loss (permanently losing majority of cluster manager nodes) in the cluster.
+
+Cluster State in an internal data structure which contains the metadata of the cluster along with other information. 
+The metadata includes details about index metadata like settings, mappings, active copies of the shards, cluster level settings, aliases, templates and data streams etc. 
+This metadata is managed by the elected cluster manager node and is essential for proper functioning of the cluster. When the cluster loses majority of the cluster manager nodes permanently, lets say 2 out of 3 cluster manager nodes are lost, then the cluster can experience data loss as there are no guarantees that latest cluster state metadata is present in the surviving nodes. Today, cluster durability is the function of the cluster manager node storage. And, persisting the state to remote provides better durability guarantees.
+
+When remote cluster state feature is enabled, the cluster metadata will be published to a remote repository configured in the cluster. Note that, currently only index metadata will be persisted to remote store in OpenSearch 2.10.
+Any time, the new cluster manager nodes are launched after disaster recovery, they will bootstrap using the latest index metadata stored in the remote repository automatically. Consequently, the data of the indices will also be restored when remote store is enabled.
```

Comment:
```
Consequently, the data of the indices will also be restored when remote store is enabled.
```

this indicates remote state and store can be enabled separately. but as of now they cannot be. the cluster settings for remote state enabled/disabled is just a kill switch

Reply:
Should we add that in the limitation : 
```
Remote cluster state cannot be enabled without remote segment/translog being configured
```